e2e/gui/src/test/java/com/epam/pipeline/autotests/GlobalSearchTest.java

```diff
@@ -383,4 +475,11 @@ public void searchAfterDeleting() {
     private GlobalSearchAO search() {
         return new NavigationMenuAO().search();
     }
+
+    @AfterTest
```

Comment:
It would be better to move this method before @Test

Reply:
Done